From hjl@nynexst.com Sun Feb 13 18:48:07 EST 1994
Article: 19356 of comp.os.linux.help
Path: bigblue.oit.unc.edu!concert!news-feed-2.peachnet.edu!darwin.sura.net!howland.reston.ans.net!europa.eng.gtefsd.com!news.ans.net!news.nynexst.com!jalod!hjl
From: hjl@nynexst.com (H.J. Lu)
Newsgroups: comp.os.linux.help
Subject: PS/2 mouse bug in Selection 1.5 (Re: TOSHIBA Laptop - Results)
Date: 12 Feb 1994 03:10:21 GMT
Organization: Nynex Science & Technology, Inc.
Lines: 80
Message-ID: <2jhhat$kqc@news.nynexst.com>
References: <2j3is0$kpm@rzsun02.rrz.uni-hamburg.de> <7341@chorus.chorus.fr> <1994Feb10.205445.26447@infodev.cam.ac.uk>
NNTP-Posting-Host: jalod.nynexst.com
X-Newsreader: TIN [version 1.2 PL2]

Martin Beckett (mgb@mail.ast.cam.ac.uk) wrote:
: The summary for installing LINUX on a toshiba 1950CT laptop

: Configuration: Tosh 1950CT / 4Mb internal ram / 8Mb RAM Card / 324 Mb disk

I have an Arima 486DX2/66.

: Slackware 1.1.1 loaded with no problems at all
: The Toshiba ram card was recognised automatically without even setting any CMOS

: The only problem was that although it looks like a PCMCIA card it doesn't
: fit in the PCMCIA slot , it goes in a hidden slot just above this - also
: the manual says to fit it upside down , wrong! 

: Xfree 2.0 works fine in mono or default vga mode with a serial mouse
: The video chipset is a WD90C24 which is not supported in VGA although I have
: been told that the WD90C30 driver works = will try this tomorrow.

Yes. It works for me.

: The trackball mouse isn't supported as far as I can see but it might emulate
: a bus mouse , ( ANyone know ? )

You can try it as PS/2 mouse. BTW, there is a bug in selection 1.5.
Here is the patch. Could someone please forward it to the right
person?

Thanks.

: In the next few days I will get an ethernet adaptor and try the networking


H.J.
-----
*** mouse.c.orig	Thu Jun 17 19:11:04 1993
--- mouse.c	Fri Feb 11 20:56:26 1994
***************
*** 154,160 ****
      {	0xe0,	0x80,	0x80,	0x00,	3	},  /* MMSeries */
      {	0xe0,	0x80,	0x80,	0x00,	3	},  /* Logitech */
      {	0xf8,	0x80,	0x00,	0x00,	5	},  /* BusMouse */
!     {   0xcc,	0x00,	0x00,	0x00,	3	}   /* PS/2 */
  };
  
  static void
--- 154,160 ----
      {	0xe0,	0x80,	0x80,	0x00,	3	},  /* MMSeries */
      {	0xe0,	0x80,	0x80,	0x00,	3	},  /* Logitech */
      {	0xf8,	0x80,	0x00,	0x00,	5	},  /* BusMouse */
!     {   0xc0,	0x00,	0x00,	0x00,	3	}   /* PS/2 */
  };
  
  static void
***************
*** 209,216 ****
  	return -1;
      }
  
!     if (mtype != P_BM && mtype != P_PS2)
      {
  	ms_setspeed(9600, mbaud, cflag[mtype]);
  	ms_setspeed(4800, mbaud, cflag[mtype]);
  	ms_setspeed(2400, mbaud, cflag[mtype]);
--- 209,224 ----
  	return -1;
      }
  
!     switch (mtype)
      {
+     case P_BM:
+       break;
+ 
+     case P_PS2:
+       write (mfd, "\364", 1);
+       break;
+ 
+     default:
  	ms_setspeed(9600, mbaud, cflag[mtype]);
  	ms_setspeed(4800, mbaud, cflag[mtype]);
  	ms_setspeed(2400, mbaud, cflag[mtype]);


